.gitignore

```diff
@@ -5,6 +5,7 @@ etc/opt
 lib/bash/rose_init_site
 doc
 venv
+metomi_rose.egg-info
```

Comment:
@kinow Should I be excluding the egg-info file?

Reply:
Yep.